docs/_documentations/che-installinfo.md

```diff
@@ -14,39 +14,101 @@ Install Che to use with Codewind or prepare to use Codewind with an existing Che
 
 ## Prerequisites
 
-1. Set up PersistentVolumes (PVs) that support both `ReadWriteOnce` (RWO) and `ReadWriteMany` (RWX) access modes and have a minimum of 1 Gi storage.
+1. Set up PersistentVolumes (PVs) that support both `ReadWriteOnce` (RWO) and `ReadWriteMany` (RWX) access modes and each have a minimum of 5 Gi storage.
    - One volume is required for Che, and two volumes are required for each Codewind workspace.
    - For Network File System (NFS) shares, set 777 permissions for the exported folders and ownership of `nobody:nogroup`.
    - Because Codewind uses `ReadWriteMany` (RWX) volumes to provide persistent storage, use NFS for storage on OpenShift 4.
 2. Ensure that the cluster can pull images from the `docker.io/eclipse` and `quay.io/eclipse` registries.
    - Both Eclipse Che and Eclipse Codewind host Docker images at these locations.
    - Many clusters have image policies that control which registries you can use to pull images. Check your cluster documentation and ensure that the cluster image pull policies permit both of these registries.
 3. Set up the ClusterRole for Codewind:
-`kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.9.0/setup/install_che/codewind-clusterrole.yaml`
+`kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.10.0/setup/install_che/codewind-clusterrole.yaml`
 
 ## Installing Che with chectl
 
+Codewind requires Eclipse Che to be installed with HTTPS in order to function properly. The following section, `Installing Che` will cover setting up Eclipse Che with TLS certificates.
```

Comment:
**Recommended rewrite:** Codewind requires Eclipse Che to be installed with HTTPS to function properly. See the [Installing Che](#installing-che) section to set up Eclipse Che with TLS certificates.